components/gitpod-protocol/src/protocol.ts

```diff
@@ -754,7 +754,9 @@ export interface PrebuiltWorkspace {
 
 export namespace PrebuiltWorkspace {
     export function isDone(pws: PrebuiltWorkspace) {
-        return pws.state === "available" || pws.state === "timeout" || pws.state === "aborted";
+        return (
```

Comment:
though `isDone` seems to be very unused after this PR, it seems to be wrong to have the `failed` state missing in here.

Reply:
If it's unused, let's remove it. It's always available in git history.

Reply:
It's API though.

Reply:
> It's API though.

What do you mean?